consumers/hidden-test-consumer/Dockerfile

```
@@ -0,0 +1,39 @@
+ENV POETRY_NO_INTERACTION=1 \
+    POETRY_VIRTUALENVS_IN_PROJECT=true \
+    POETRY_HOME=/home/codesirius/.poetry
+
+# add poetry to PATH
+ENV PATH="${PATH}:${POETRY_HOME}/bin"
+
+# create a non-root user
+RUN adduser --disabled-password codesirius
+
+# install poetry
+RUN apt-get update && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)